scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Connection.java

```diff
@@ -105,6 +118,32 @@ public Connection(final SessionTicket sessionTicket, SessionId sessionId) {
 		this.peerAddress = null;
 		this.autoResumptionTimeout = null;
 		this.sessionListener = null;
+		this.maxDeferredProcessedApplicationDataMessages = 0;
+		this.deferredApplicationData = EMPTY;
```

Comment:
Why setting maxDeferredProcessedApplicationDataMessages to 0 and deferredApplicationData to empty ?

Reply:
The "ticket connection" is only used to return the ticket for the session resumption. the connection is never really used as connection.